flatpak/ldopts.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/makefile b/makefile
+index c5cedbc0..727320e5 100644
+--- a/makefile
++++ b/makefile
+@@ -60,7 +60,7 @@
+ # ARCHOPTS_OBJC =
+ # ARCHOPTS_OBJCXX =
+ # OPT_FLAGS =
+-# LDOPTS =
++LDOPTS = -Wl,-s
+ 
+ # USE_SYSTEM_LIB_ASIO = 1
+ # USE_SYSTEM_LIB_EXPAT = 1
```

Comment:
Can you do this by specifying it on the make command line, or by putting the options you need in a file and setting `PREFIX_MAKEFILE` in the make command line?

Reply:
Yes, probably the make command line is the cleaner of the two.  This is an external patch applied on the fly during the build step, just to point it out